content/contributing-terms.en.md

```diff
@@ -6,9 +6,9 @@ title: "Contributing terms"
 
 ## Tracking new terms
 
-Tracking terms is done by _declaring_ them and the service they are associated with. Such a declaration is achieved by editing JSON files in the [`declarations`](./declarations) folder.
+Tracking terms is done by _declaring_ them and the service they are associated with. Such a declaration is achieved by editing JSON files in the [`declarations`](https://github.com/OpenTermsArchive/contrib-declarations/tree/main/declarations) folder.
```

Comment:
This is misleading: the content dates back from when there was a single collection, and thus a single `declarations` folder. Nowadays, each collection has its own `declarations` folder, so these instructions should start by explaining how to choose a collection, and not have any direct link to de `declarations` folder of any specific collection.

I will still accept this in this pull request, as it is better to have a link to a specific collection than a broken link 👍